packages/zoe/src/contracts/atomicSwap.js

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable no-use-before-define */
 import harden from '@agoric/harden';
 
-import { makeZoeHelpers } from './helpers/zoeHelpers';
+// Should be imported from '@agoric/zoe-contractSupport'
```

Comment:
I think we cannot have any upper case letters in npm packages, so this would probably be `@agoric/zoe-contract-support`

Should we say, "Eventually will be importable from..." instead?

Reply:
done